### ISSUE TITLE
Check for running deployment

### ISSUE THREAD

**Issue by janatzend** [zend-patterns/ZendServerSDK#79]
A deployment which is being started before another deployment has been finished, can cause an unpredictable behavior in Zend Server. It would be nice to have a flag for the 'deployment' command to check for running deployments before executing the actual deployment. If 'yes', just stop the process (block the deployment)

**Comment by slaff** [comment]
e57eac29acb4bcfa9de4b8fb6f7c281dc2cc7c64 contains the initial changes for this feature request. 
`--safe` will check if there is on-going deployment and if yes will exit with error
`--safe` in combination with `--wait` will play it safe and wait for the on-going to finish before starting the new deployment.

**Comment by slaff** [comment]
@janatzend Please, check if this is working for you and if yes, close the feature request.
